README.md

```diff
@@ -1,13 +1,22 @@
-# Happy Thoughts
+# Happy Thoughts 💌
```

Comment:
Great readme, you have really explained your app and how you approach your problem :)